core/commands/active.go

```diff
@@ -22,7 +22,7 @@ Lists running and recently run commands.
 		res.SetOutput(req.InvocContext().ReqLog.Report())
 	},
 	Options: []cmds.Option{
-		cmds.BoolOption("v", "verbose", "print more verbose output"),
+		cmds.BoolOption("v", "verbose", "print more verbose output").Default(false),
```

Comment:
They're not always false. We could specify only when default is true? cc @jbenet